src/test/java/com/epam/training/center/hw2/ex1/AbstractPage.java

```diff
@@ -0,0 +1,28 @@
+package com.epam.training.center.hw2.ex1;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+
+import java.util.concurrent.TimeUnit;
+
+public abstract class AbstractPage {
```

Comment:
You should have only 1 abstract class for 2 hw :)

Moreover, "Page" in name can mislead. You can use it in third task where PageObject required. But here it is mislead. Use standart "BaseTest" pls